pkg/lang/python/plugin_expose.go

```diff
@@ -210,23 +210,23 @@ func (h *restAPIHandler) handleFile(f *core.SourceFile) (*core.SourceFile, error
 			continue
 		}
 
-		appName = app.Identifier.Content(f.Program())
+		appVarName = app.Identifier.Content(f.Program())
 		h.RootPath = app.RootPath
 
 		gwSpec := gatewaySpec{
 			FilePath:   f.Path(),
-			AppVarName: cap.ID,
+			AppVarName: appVarName,
```

Comment:
isnt this going to result in validation to fail again. On line 130 we call 

```python
		gw := core.NewGateway(spec.AppVarName)
```

so then wont this not match our ID? other gatewayspecs have an id field so we can track it so i dont think this prevents the warning from popping up

Reply:
yeah, I just went back and added `gatewayId` to the spec and updated this line to use that value.